internal/page/page.go

```diff
@@ -63,6 +60,15 @@ func WithLimit(ctx context.Context, limitStr string) context.Context {
 	return context.WithValue(ctx, limitCtxKey, limit)
 }
 
+// WithToken returns context copy with token value.
+func WithToken(ctx context.Context, token string) context.Context {
+	logger := zerolog.Ctx(ctx)
+	if token == "" {
+		logger.Warn().Msg("token is empty, listing first page")
```

Comment:
We might want to use `logger.Info()` or even `logger.Debug()` here I guess, this not any unexpected state, right? we do support requests without token, so there is no reason to Warn about hitting supported case IMO :)

Reply:
Yes, I guess info will be more accurate, changing :)